crypto/x509/x509_acert.c

```diff
@@ -326,3 +326,76 @@ const STACK_OF(X509_EXTENSION) *X509_ACERT_get0_extensions(const X509_ACERT *x)
 {
     return x->acinfo->extensions;
 }
+
+int X509_verify_acert(X509_STORE_CTX *ctx, X509_ACERT *acert)
```

Comment:
X509_STORE_CTX_ACERT_verify would make more sense as a name ... I know there is an X509_verify that takes just X509_STORE_CTX as an arg - but there is also X509_STORE_CTX_verify too which is closer to this. 

I'd also argue whether or not this should even exist as a function and whether or not it should be combined by having extra items added into the X509_STORE_CTX and have the processing there supplemented - especially when reading the documentation for the function - as it assumes the user has done the standard verify first.

Reply:
I actually intentionally did not touch `X509_STORE_CTX` just because I know that struct is so foundational, but I would be happy to add an `acert` pointer field to it, and just verify that as part of the entire PKI path. And actually I think that makes sense, because I think there are some X.509 requirements on what public key certificates can issue attribute certificates.

Reply:
Do you think it would be good for there to be a `STACK_OF(X509_ACERT)` instead of a single pointer to one? Then we could validate multiple attribute certificates in a single pass. It just complicates the implementation a little bit, since we would have to identify which attribute certificate had the verification issue, if any.

Reply:
Actually, on that note, should such a change be a part of this PR or part of the implementation PR?